MYabrv.bib

```diff
@@ -145,13 +147,14 @@ @String{SBST
 @String{SEAS = "Proc.\ Int'l Workshop on Software Engineering for Automotive Systems (SEAS)"}
 @String{SF = "Proc.\ Int'l Workshop on Software Factories (SF)"}
 @String{SPLAT = "Proc.\ Workshop on Software Engineering Properties of Languages and Aspect Technologies (SPLAT)"}
-@String{SPLatools = "Proc.\ Workshop on Software Product Line Analysis Tools (SPLat)"}
+@String{SPLATools = "Proc.\ Workshop on Software Product Line Analysis Tools (SPLat)"}
```

Comment:
the ending "ools" is to avoid a name clash. but I do not get why you want to change the capitalization here (see end of the line).